snap/snapcraft.yaml

```diff
@@ -72,13 +72,43 @@ parts:
       snapcraftctl set-version $mir_version-snap$recipe_version
       if echo $mir_version | grep -e '+dev' -e '~rc' -q; then snapcraftctl set-grade devel; else snapcraftctl set-grade stable; fi
     stage-packages:
-      - mir-graphics-drivers-desktop
```

Comment:
Are we *sure* no one is using Mir-on-Wayland?

Reply:
Yes, there is no plug to support that:

```yaml
    plugs:
      - login-session-control
      - x11
```